### PR TITLE
Update VNC instructions

### PR DESCRIPTION
**Pull request by robrap**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -688,9 +688,9 @@ you can connect to the container running it via VNC.
 | Chrome (via Selenium)  | vnc://0.0.0.0:15900  |
 +------------------------+----------------------+
 
-On macOS, enter the VNC connection string in Safari to connect via VNC. The VNC
-passwords for both browsers are randomly generated and logged at container
-startup, and can be found by running ``make vnc-passwords``.
+On macOS, enter the VNC connection string in the address bar in Safari to
```
Comment (robrap) [inline README.rst:691]: I know this may seem stupid, but I always think it is going to be more complicated than it is.

File: README.rst [hunk 2]
```diff
@@ -688,9 +688,9 @@ you can connect to the container running it via VNC.
...
+connect via VNC. The VNC passwords for both browsers are randomly generated and
+logged at container startup, and can be found by running ``make vnc-passwords``.
 
 Most tests are run in Firefox by default.  To use Chrome for tests that normally
```
Comment (robrap) [inline README.rst:695]: Also, is this no longer true?

**Review by robrap — COMMENTED** [review]
@jmbowman: Some starter comments...

File: README.rst (hunk 2 above)
Comment (robrap) [reply README.rst:695]: Looks like it is still Firefox, so ignore this comment. :)

Comment (jmbowman) [reply README.rst:695]: Right, a switch to Chrome is in the works but not done yet; need to fix some tests that fail under it first.

**Review by jmbowman — APPROVED** [review]

**Comment by robrap** [comment]
@jmbowman: Some questions:
1. What is appveyor?
2. Do we need special access to rerun it?

It seemed to fail with the following:
> Build execution time has reached the maximum allowed time for your plan (60 minutes).

**Comment by jmbowman** [comment]
AppVeyor is the service we're using to test devstack on Windows.  Our plan recently reverted from a trial to the free-for-open-source tier, I'll create a separate PR to update our test script accordingly.  The trial included beta-level support for running Linux containers on Windows, but that isn't really helping us until we figure out how to allocate more RAM to the underlying Linux VM; I'll just stub out the code that tries to launch the VM for now.

In the meantime, don't consider that failure a blocker for merging this PR.

**Pull request merged**